.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macOS-latest ]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Prepare Ubuntu
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y pandoc ffmpeg
```

Comment:
I've recently seen the flag `--no-install-recommends`, which might reduce the number of unnecessarily installed packets?

I haven't tried it yet, though, so I don't know whether it actually makes any difference.

Reply:
My hope would be that it never install recommended package,  but just recommends them. But I will check.

Reply:
I was wrong, there is [required, recommended, suggested](https://ubuntu.com/blog/we-reduced-our-docker-images-by-60-with-no-install-recommends) packages and only the suggested will not be installed.
So, it seems to be indeed a good idea to add `--no-install-recommends`, which I did now.

Reply:
Good to know!